integreat_cms/cms/models/external_calendars/external_calendar.py

```diff
@@ -31,6 +31,28 @@ class ExternalCalendar(AbstractBaseModel):
         ),
     )
     errors = models.CharField(verbose_name=_("import errors"), default="", blank=True)
+    created_by = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="created_by",
```

Comment:
```suggestion
```
We probably don't want that?
(related name specifies how to follow this relationship from the user model, so in this case it would be possible to get all external calendars of a user with `user.created_by`, which seems confusing)

Reply:
I agree that this is confusing, but since both fields call to the User model and both of them would have the same default reverse accessor name, it's necessary to distinguish them. As far as I understand it's necessary for at least one of the two fields to have a related_name